### PR TITLE
add bookmark on hover

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes

Add Bookmark on Hover to NFT Previews

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/80802871/fd9d6cdd-5d25-4c7a-aa62-f645cb3207f7

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/web/src/components/NftPreview/NftPreviewBookmarkLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  }, [admireToken, decodedTokenName, info, query, route, showModal, token.dbid, token.id, track]);
+  const handleRemoveBookmark = useCallback(() => {
+    if (
+      !token.viewerAdmire?.dbid ||
+      query.viewer?.__typename !== 'Viewer' ||
+      !query.viewer?.user?.dbid
+    ) {
+      return;
+    }
+    track('NFT hover: Clicked remve from bookmarks', {
```
Comment (Robinnnnn) [inline apps/web/src/components/NftPreview/NftPreviewBookmarkLabel.tsx:108]: smol typo

Comment (kaitoo1) [reply apps/web/src/components/NftPreview/NftPreviewBookmarkLabel.tsx]: good catch fixed!
